py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/error.py

```diff
@@ -13,18 +13,6 @@ def __init__(self, filename):
         self.message = \
             "Unable to deduce Configuration Type from filename %s" % filename
 
-class ConversionExceptioon(GTFSIngestException):
-    """
-    General Error to be used durring Conversion Failures
-    """
-    pass
-
-class S3Exception(GTFSIngestException):
-    """
-    General Error to be chained with boto3 Errors
-    """
-    pass
-
```

Comment:
we don't need these. all exceptions in the conversion methods are going to be logged, not raised up, since they are handled in a thread, and we don't want them to kill the process.